src/models/sql/employees.model.js

```diff
@@ -78,15 +78,85 @@ const tableOptions = {
 
 const employees = connection.define(TABLE_NAME, fields, tableOptions);
 
-const save = async (dbDate) => {};
+const save = async (dbData) => {
+  dbData.employeeId = uuid.v4()
+  return employees.create(dbData);
+};
 
-const update = async (employeeId, dbData) => {};
+const update = async (req, res) => {
+  const id = req.params.id;
+  employees.update(req.body, {
+    where: {
+      employeeId: id
+    }})
+    .then(num => {
+      if (num == 1) {
+        return res.status(200).send({ 
```

Comment:
for the return rsp we can api-rsp module

Reply:
Have implemented the api-rsp initially in Products.Controller 'create' method . If you can confirm whether its correct, will go ahead and implement across places.

Reply:
Yes, It's correct. The same way we can use for error response a well. You can check those function definitions as well.